packages/bumbag/src/ColorMode/InitializeColorMode.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 const noflash = `(function() { try {
   var mode = localStorage.getItem('bb.mode');
```

Comment:
I don't have a custom `localStoragePrefix` set personally, but this code won't work as expected for those who do.

Perhaps that prefix should be saved somewhere so that this little hack can retrieve it here and correctly apply the colour mode?

Reply:
Yeah, you are right. I don't think local storage is the best solution here anyway. I think this needs more thinking & a refactor.